db/post_migrate/20210528003603_fix_badge_image_avatar_upload_security_and_acls.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class FixBadgeImageAvatarUploadSecurityAndAcls < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def up
+    upload_ids = DB.query_single(<<~SQL
+      SELECT image_upload_id
+      FROM badges
+      WHERE image_upload_id IS NOT NULL
```

Comment:
actually, in that case, wouldn't it be better to join uploads where secure is true, to just have ids that should be updated and then in later statements use only

`WHERE id IN (:upload_ids)` ?

Reply:
Ah yes I see what you mean, will update